### PR TITLE
ExtractDomainFromUrlFormat-packagify

### PR DESCRIPTION
**Pull request by reutshal**
## Status
Ready

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [X] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by yardensade** [comment]
Extract*, please change file name

File: Scripts/ExstractDomainFromUrlFormat/ExstractDomainFromUrlFormat.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+commonfields:
+  id: ExtractDomainFromUrlAndEmail
+  version: -1
+name: ExtractDomainFromUrlAndEmail
+script: ''
+type: python
```
Comment (yardensade) [inline Scripts/ExstractDomainFromUrlFormat/ExstractDomainFromUrlFormat.yml]: Seems like subtype is missing here

File: Scripts/ExstractDomainFromUrlFormat/ExstractDomainFromUrlFormat.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            the_input = proofpoint_get_original_url(the_input)
+        # Not ATP Link or Proofpoint URL so just unescape
+        else:
+            the_input = unescape_url(the_input)
+        is_url = domain = get_tld(the_input, fail_silently=True)
+
+    # Extract domain itself from a potential subdomain
+    if domain_from_mail or not is_url:
+        full_domain = 'https://'
+        full_domain +=  domain_from_mail if domain_from_mail else the_input
```
Comment (amshamah419) [inline Scripts/ExstractDomainFromUrlFormat/ExstractDomainFromUrlFormat.py]: Remove the extra space behind `=`

Comment (reutshal) [reply Scripts/ExstractDomainFromUrlFormat/ExstractDomainFromUrlFormat.py]: thanks. I'll run the linting test before pushing again

**Pull request closed without merging**